src/panels/NightSkyPanel/tabs/NightSkyTimeTab.tsx

```diff
@@ -0,0 +1,194 @@
+      </Text>
+      <Text my={'md'} style={{ flexGrow: 12 }}>
+        UTC: {timeLabel}
+      </Text>
+      <Divider mt={5} />
+      <Text my={'md'} size={'xl'}>
+        Jumps
+      </Text>
+      <Stack mb={'lg'}>
+        <Group gap={'xl'}>
+          <Group>
+            <Button
+              onClick={() => luaApi?.action.triggerAction('os.time.siderealDayIncrease')}
+            >
+              + sidereal day
+            </Button>
+            <Button
+              onClick={() => luaApi?.action.triggerAction('os.time.siderealDayDecrease')}
+            >
+              - sidereal day
+            </Button>
+            <Button
+              onClick={() => luaApi?.action.triggerAction('os.time.siderealWeekIncrease')}
+            >
+              + sidereal week
+            </Button>
+            <Button
+              onClick={() => luaApi?.action.triggerAction('os.time.siderealWeekDecrease')}
+            >
+              - sidereal week
+            </Button>
+          </Group>
+          <Group>
+            <Button
+              onClick={() => luaApi?.action.triggerAction('os.time.SolarDayIncrease')}
+            >
+              + solar day
+            </Button>
+            <Button
+              onClick={() => luaApi?.action.triggerAction('os.time.SolarDayDecrease')}
+            >
+              - solar day
+            </Button>
+            <Button
+              onClick={() => luaApi?.action.triggerAction('os.time.SolarWeekIncrease')}
+            >
+              + solar week
+            </Button>
+            <Button
+              onClick={() => luaApi?.action.triggerAction('os.time.SolarWeekDecrease')}
+            >
+              - solar week
+            </Button>
+          </Group>
+        </Group>
+        <Divider mt={5} />
+        <Text size={'xl'}>Diurnal Motion</Text>
+        <Group>
+          <Button
+            onClick={() => {
+              luaApi?.time.interpolatePause(false);
+              luaApi?.time.interpolateDeltaTime(1);
+            }}
+          >
+            Realtime
+          </Button>
+          <Button
+            onClick={() => {
+              luaApi?.time.interpolatePause(false);
+              luaApi?.time.interpolateDeltaTime(60);
+            }}
+          >
+            60x (1 min/sec)
+          </Button>
+          <Button
+            onClick={() => {
+              luaApi?.time.interpolatePause(false);
+              luaApi?.time.interpolateDeltaTime(120);
+            }}
+          >
+            120x (2 min/sec)
+          </Button>
+          <Button
+            onClick={() => {
+              luaApi?.time.interpolatePause(false);
+              luaApi?.time.interpolateDeltaTime(300);
+            }}
+          >
+            300x (5 min/sec)
+          </Button>
+
+          <Button onClick={() => luaApi?.time.togglePause()}>Play / Pause</Button>
+        </Group>
+      </Stack>
+      <Divider mt={5} />
+      <Paper my={'lg'} shadow={'xs'} p={'xl'}>
```

Comment:
Simplify styling as much as possible, and try to use exising sizes. Much easier to maintain
```suggestion
      <Divider my={'md'} />
      <Paper p={'md'}>
```